packages/app/chrome/test/chrome.test.ts

```diff
@@ -0,0 +1,34 @@
+import { resolve } from "path";
```

Comment:
`packages/app/chrome` will be disappearing soon. Might be nice to have this in `test/*`

Reply:
`packages/app/test`?

Reply:
Nah just `test/`

Reply:
Like at the root of the repo. These are tests for the overall build of the repo afterall.